libs/sdk-ui-dashboard/src/presentation/filterBar/filterBar/DefaultFilterBar.tsx

```diff
@@ -94,24 +95,28 @@ export function DefaultFilterBar(props: IFilterBarProps): JSX.Element {
                 {dateFilterMode === "hidden" ? (
```

Comment:
The date filter should still be visible in edit mode, regardless of dateFilterMode. dateFilterMode should only be applied/respected in view mode.

Reply:
fixed in selectEffectiveDateFilterMode